### ISSUE TITLE
Able to access QR code or barcode via photo library in device

### ISSUE THREAD

**Issue by khajeemart** [doug-m/CollectorAuroraTesting#258]
### Issue
QR code or barcode cannot access via photo library in device (iPhone).

### Map and Credentials

* Group Name:
* Map Name:

### Steps to Reproduce
1. Open Collector for ArcGIS
2. Select login "Sign In With ArcGIS Enterprise"
3. Select "Scan Code" 
4. Should scan via camera or access via photo library in device 

### Device Information
* Device: iPhone 8 plus
* OS Version: 13.2.3

**Comment by colinlawr** [comment]
@khajeemart The ability to scan a QR code to sign into Portal is coming in an upcoming beta. 

Can you clarify the part about accessing a photo on your device? Are you referring to a QR stored as a photo on your device?

**Comment by khajeemart** [comment]
> @khajeemart The ability to scan a QR code to sign into Portal is coming in an upcoming beta.
> Can you clarify the part about accessing a photo on your device? Are you referring to a QR stored as a photo on your device?

Yes, I referring to QR code stored as a photo on my device. We can select scan QR code via camera or select QR code photo on device.

**Comment by colinlawr** [comment]
@khajeemart Can you outline your use case for scanning a QR code from the phones library? Why would this be more beneficial to you or your organization? 

Looking into this a bit it seems it requires a 3rd party app to 'scan' from a photo on your device.

**Comment by khajeemart** [comment]
Outline: 
After generate QR code portal, you can save QR code photo on device.
1. Open Collector for ArcGIS
2. Select login "Sign In With ArcGIS Enterprise"
3. Select "Scan Code"
4. You can scan QR code via camera or select QR code (photo) via photo library on device.
(Example: When you add friend in Line app by QR code, we can select scan via camera or  )

I think this is easy for field surveying because we no carry many thing, such as paper consist of QR code.
